<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPassword</class>
 <widget class="QWidget" name="ModifyPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModifyPassword</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ModifyPassword{ background: rgba(32, 80, 96, 100); }
QLineEdit { border-radius: 3px; }</string>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>156</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 5px;
}

QPushButton:hover { background-color: #0080FF; }</string>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>156</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>4</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: white;
  background-color: #27a9e3;
  border-width: 0px;
  border-radius: 5px;
}

QPushButton:hover { background-color: #0080FF; }

QPushButton:disabled{
 background-color: #a6a6a6;
}</string>
   </property>
   <property name="text">
    <string>Salva</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Conferma Nuova Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Nuova Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password Attuale</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color: white; }</string>
   </property>
   <property name="text">
    <string>Cambia Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
